handlebars/src/main/java/com/github/jknack/handlebars/internal/Partial.java

```diff
@@ -171,18 +196,37 @@ protected void merge(final Context context, final Writer writer)
         }
 
       }
+      context.data("callee", this);
```

Comment:
This is something internal and it shouldn't be called `callee`. Let's create a name which is hard to override by users like `Context. INVOCATION_STACK` or/and by prefixing the class name: `com.github.jknack.handlebars.internal.Partial.calle`